### ISSUE TITLE
Add SplitArchive support to ZipArchive

### ISSUE THREAD

**Issue by dos-ise** [dotnet/runtime#102306]
Now that dotnetzip is no longer maintained. We want to migrate to System.IO.Compression
Unfortunately we are blocked by the missing support of SplitArchive.

Are there any plans to add support for this?

[DotNetZip](https://github.com/haf/DotNetZip.Semverd)
[SplitArchives](https://github.com/eropple/dotnetzip/blob/master/Zip%20Tests/SplitArchives.cs)

Oh and the other missing feature is  the support of password proteced archives. :)
[Password support](https://github.com/dotnet/runtime/issues/1545)

**Comment by karelz** [comment]
Triage: We might look into implementation of READING split archives - to assess feasibility.
Opening such archives for Creation or Update would be out of scope.

Is that what you expected @dos-ise ?

Note: This is not a promise we will do it / accept such contribution, just that we are willing to investigate its feasibility.

**Comment by dos-ise** [comment]
@karelz Yes, that sounds absolutely reasonable — even just supporting read access for split archives would already be a fantastic improvement and simplify many existing workflows. Totally understand that creation and update are out of scope for now. Thanks for looking into feasibility!
